.prettierrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "trailingComma": "all",
   "tabWidth": 2,
+  "printWidth": 60,
```

Comment:
To change these parameters there should be a mutual agreement

Reply:
Ok, let's talk about it. According many articles use more than 100 characters in no useful (this is an example https://prettier.io/docs/en/options.html#print-width). we could change it to 80 chars per line, I think no more of that value.